src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -45,10 +44,66 @@
  * {@code Runtime} that allows the application to interface with
  * the environment in which the application is running. The current
  * runtime can be obtained from the {@code getRuntime} method.
+ *
+ * <p>An application cannot create its own instance of this class.
+ *
+ * <h2><a id="shutdown">Shutdown Sequence</a></h2>
+ *
+ * <p>The Java Virtual Machine initiates the <i>shutdown sequence</i> in response
+ * to one of several events:
+ * <ol>
+ * <li>when the number of {@linkplain Thread#isAlive() live} non-daemon threads drops to zero
```

Comment:
Can we somehow forward-reference the implNote about DestroyJavaVM here?

Reply:
added